Dockerfile

```diff
@@ -5,10 +5,12 @@ WORKDIR /bot
 COPY Pipfile Pipfile.lock /bot
 
 RUN apt-get update && apt-get -y install cron && rm -rf /var/lib/apt/lists/*
+RUN apt-get install -yq tzdata && \
+    ln -fs /usr/share/zoneinfo/America/New_York /etc/localtime && \
+    dpkg-reconfigure -f noninteractive tzdata
```

Comment:
Instead of running dpkg chained off of the other command, it would be more ideal to separate the layer into two and use another `RUN` for dpkg.